### PR TITLE
Add guidance on availability, communicating and applying.

### PR DESCRIPTION
**Pull request by brainwane**
Add a heads-up that we won't have enough slots to take all interested Outreachy applicants, and add tips on publishing and strengthening internship applications (including by being a good citizen in shared communication spaces).

### CODE REVIEW

File: outreachy-round-13.md [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
 
 If you're [in a group underrepresented in open source
 communities](https://wiki.gnome.org/Outreachy#Eligibility) and you're
 interested in contributing to Zulip during a paid apprenticeship,
 please apply by October 17th!
 
+(We already have more applicants than we have mentors available; if
+you're just hearing about Outreachy now, and you aren't already
+interested in Zulip, consider [one of the other mentoring
+organizations](https://wiki.gnome.org/Outreachy/2016/DecemberMarch#Participating_Organizations).)
```
Comment (timabbott) [inline outreachy-round-13.md:15]: I would say "aren't specifically interested in Zulip" -- I don't think we should push people away who are either especially interested in Zulip or feel confident in their skills.  I also don't like framing it as "We already have more applicants than we have mentors available;" -- that'll be true with every organization at this point.  I think we should instead frame it as "We've already received many applications"

File: outreachy-round-13.md [hunk 2]
```diff
@@ -83,15 +89,60 @@ be skilled in writing in English for a non-academic audience, and to
...
+1) you ask a question
+2) someone directs you to a document
+3) you go read that document, and try to use it to answer your question
+4) you find you are confused about a new thing
+5) you ask another question
+6) now that you have shown that you have the ability to read, think, and
+learn new things, someone has a longer talk with you to answer your new
+specific question
+7) you and the other person collaborate to improve the document that you
+read in step 3 :-)
```
Comment (timabbott) [inline outreachy-round-13.md:136]: can you add a link to that Akamai post?  I think we should encourage folks to search our documentation and struggle a little bit before we give advice with step 1 being "ask a question".

File: outreachy-round-13.md (hunk 1 above)
Comment (brainwane) [reply outreachy-round-13.md:15]: You're right of course. Changing.

File: outreachy-round-13.md (hunk 2 above)
Comment (brainwane) [reply outreachy-round-13.md:136]: Got it. Fixed.

**Comment by brainwane** [comment]
Ready for fresh review.

**Comment by timabbott** [comment]
Merged, thanks @brainwane!

**Pull request merged**